src/main/java/uk/gov/hmcts/reform/blobrouter/exceptions/InvalidSasTokenException.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.reform.blobrouter.exceptions;
+
+public class InvalidSasTokenException extends RuntimeException {
+
+    public static final String EXPIRY_NOT_FOUND = "Invalid SAS, the SAS expiration time parameter not found.";
```

Comment:
Suggestion: as this exception accepts message, it would be better for this specific message to be defined where it's provided.

Reply:
that exception may called from different locations?

Reply:
Currently, it's only created in one place. If there's another usage, the context (and the message) may be different - otherwise we would not need to accept `message` parameter in the constructor.  It felt a bit unnatural to me that this exception defines a constant that is passed by a different class to the constructor of this class.

This is a suggestion, to up to you.